### PR TITLE
Added some explanation to vmctl docs

### PR DESCRIPTION
**Pull request by denisgolius**
Added explanation that vmctl do not make snapshot from Prometheus when run migration data

### CODE REVIEW

File: docs/vmctl.md [hunk 1]
```diff
@@ -274,12 +274,12 @@ You may find useful a 3rd party solution for this - https://github.com/jonppe/in
 ## Migrating data from Prometheus
 
 `vmctl` supports the `prometheus` mode for migrating data from Prometheus to VictoriaMetrics time-series database.
-Migration is based on reading Prometheus snapshot, which is basically a hard-link to Prometheus data files.
+Migration is based on reading Prometheus snapshot, which is basically a hard-link to Prometheus data files. 
 
 See `./vmctl prometheus --help` for details and full list of flags.
```
Comment (hagen1778) [inline docs/vmctl.md:279]:
```suggestion
See `./vmctl prometheus --help` for details and full list of flags. Also see Prometheus related articles [here](#articles).
```

**Review by hagen1778 — CHANGES_REQUESTED** [review]
Thanks for doing this! I think we can slightly simplify docs, see my comments.

**Comment by denisgolius** [comment]
I agree with you.

**Review by hagen1778 — APPROVED** [review]

**Comment by hagen1778** [comment]
Thank you!

**Pull request merged**